packages/address/src/AddressFormatter.ts

```diff
@@ -44,6 +44,19 @@ export default class AddressFormatter {
     return countries;
   }
 
+  async getProvinceName(
```

Comment:
Can we please call this `getZoneName`, to match the naming conventions that are used for this field in the backend. "Province" is a locale specific name, while zone is more agnostic (e.g. in the USA they're States not Provinces).

Reply:
@BPScott 
Thank you for your input, BPScott.
Your suggestion makes sense and I have changed the method name to getZoneName as you recommended.
Here is the commit:
https://github.com/Shopify/quilt/pull/2662/commits/fdf720d25ee74d770213f4a4d089d597878c024c